src/Ztm.Zcoin.Rpc/ZcoinRpcClient.cs

```diff
@@ -411,25 +421,26 @@ static ushort ToNative(PropertyType type)
 
         async Task<ExodusTransaction> TryDecodeExodusTransaction(Transaction transaction)
         {
-            var infomation = await GetExodusTransactionAsync(transaction.GetHash(), CancellationToken.None);
-            if (infomation == null)
+            ExodusTransactionInformation infomation;
+            try
+            {
+                infomation = await GetExodusTransactionAsync(transaction.GetHash(), CancellationToken.None);
+            }
+            catch (Exception) // lgtm [cs/catch-of-all-exceptions]
```

Comment:
This is dangerous due to it will also include other errors than transaction is not Exodus transaction.